packages/mask/src/plugins/ENS/base.ts

```diff
@@ -0,0 +1,40 @@
+import type { Plugin } from '@masknet/plugin-infra'
+import { NetworkPluginID } from '@masknet/web3-shared-base'
+import { ChainId } from '@masknet/web3-shared-evm'
+import { ENS_PluginID } from './constants'
+import { languages } from './locales/languages'
+
+export const base: Plugin.Shared.Definition = {
+    ID: ENS_PluginID,
+    name: { fallback: 'ENS' },
+    description: {
+        fallback: '',
+    },
+    publisher: { name: { fallback: 'Mask Network' }, link: 'https://mask.io/' },
+    enableRequirement: {
+        architecture: { app: true, web: true },
+        networks: {
+            type: 'opt-out',
+            networks: {},
+        },
+        target: 'stable',
+        web3: {
+            [NetworkPluginID.PLUGIN_EVM]: {
+                supportedChainIds: [
+                    ChainId.Mainnet,
+                    ChainId.BSC,
+                    ChainId.Matic,
+                    ChainId.Arbitrum,
+                    ChainId.xDai,
+                    ChainId.Fantom,
+                    ChainId.Avalanche,
+                    ChainId.Aurora,
+                    ChainId.Harmony,
```

Comment:
ENS only servers on Ethereum Mainnet.

```suggestion
                    ChainId.Mainnet,
```